tests/accessor/accessor_requisite_entire_buffer.cpp

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide checks that ranged accessor still creates a requisite for the entire
+//  underlying buffer
+//
+*******************************************************************************/
+
+#include "../common/disabled_for_test_case.h"
+#include "../common/get_cts_object.h"
+#include "catch2/catch_test_macros.hpp"
+
+#include "accessor_common.h"
```

Comment:
Some of these header files use features not supported by certain implementations. Typical way to fix this issue is:

```c++
// FIXME: re-enable when <FEATURE X> is implemented
#if !SYCL_CTS_COMPILING_WITH_HIPSYCL && !SYCL_CTS_COMPILING_WITH_COMPUTECPP
// includes like 
// #include "accessor_common.h"
#endif
```

Reply:
Thank you, done